src/routes/events/_events.js

```diff
@@ -2,7 +2,7 @@ import fs from "fs";
 import path from "path";
 import frontMatter from "front-matter";
 import marked from "marked";
-import { timeParse, timeFormat } from "d3-time-format";
+import { utcParse, utcFormat } from "d3-time-format";
```

Comment:
The non UTC version was actually displaying an incorrect event date for all events! Caught it because the workshop is Oct 1 and it was displaying as September 30.